src/components/list-broadcasts/ListBroadcasts.jsx

```diff
@@ -0,0 +1,15 @@
+import { BroadcastBox } from "components";
+import { List, ListItem } from "@mui/material";
+
+export const ListBroadcasts = () => {
+  return (
+    <List sx={{ padding: "0 0 2rem 0" }}>
```

Comment:
This was only one line style that's why I didn't create a new file for that. Because if I will create then I have to increase the specificity of the class name to apply the styles.
I thought instead of that let's write it here.